src/Mod/Sketcher/Gui/DrawSketchHandlerSplitting.h

```diff
@@ -54,7 +54,10 @@ class SplittingSelection : public Gui::SelectionFilterGate
             const Part::Geometry *geom = Sketch->getGeometry(GeoId);
             if (geom->getTypeId() == Part::GeomLineSegment::getClassTypeId()
                 || geom->getTypeId() == Part::GeomCircle::getClassTypeId()
-                || geom->getTypeId() == Part::GeomArcOfCircle::getClassTypeId()) {
+                || geom->getTypeId() == Part::GeomEllipse::getClassTypeId()
+                || geom->getTypeId() == Part::GeomArcOfCircle::getClassTypeId()
+                || geom->isDerivedFrom(Part::GeomArcOfConic::getClassTypeId())

```

Comment:
idem as below.

Reply:
I'll push a commit.

Off-topic: first time I hear this word "idem".

Reply:
:)